api/groups/baseAccountsGroup.go

```diff
@@ -58,8 +63,8 @@ func getTransactions(c *gin.Context) ([]data.DatabaseTransaction, int, error) {
 
 // GetAccount returns an accountResponse containing information
 // about the account correlated with provided address
-func GetAccount(c *gin.Context) {
-	account, status, err := getAccount(c)
+func (ag *accountsGroup) GetAccount(c *gin.Context) {
```

Comment:
Wondering if we still need these methods exported.

Reply:
made them unexported